pages/3.1/_index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Jakarta Server Pages 3.1 (under development)"
+date: 2021-10-15
+summary: "Release for Jakarta EE 10"
+---
+Jakarta Server Pages defines a template engine for web applications that supports mixing of textual content
+(including HTML and XML) with custom tags, expression language, and embedded Java code, that gets compiled
+into a Jakarta Servlet.
+
+* [Jakarta Server Pages 3.1 Release Record](https://projects.eclipse.org/projects/ee4j.jsp/releases/3.1.0)
+  * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
```

Comment:
This should be updated to Jakarta EE 10 with a blank url reference (or extrapolate to the expected location for the Jakarta EE 10 release plan).

Reply:
Done.